src/containers/technology/TechnologySection.js

```diff
@@ -18,20 +18,33 @@ import * as Constants from './Constants';
  */
 
 const Content = styled.div`
-  margin-bottom: 150px;
-  margin-top: 150px;
+  margin: 80px 0 100px 0;
+  width: 100%;
+
+  @media only screen and (min-width: 768px) {
+    margin: 150px 0;
+  }
 `;
 
 const SectionHeaderWrapper = styled.div`
-  width: 500px;
+  width: 100%;
+
+  @media only screen and (min-width: 768px) {
+    width: 50%;
+  }
 `;
 
 const TechTilesWrapper = styled.div`
   display: flex;
   flex: 1 0 auto;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: space-between;
+  justify-content: center;
+  max-width: 1020px;
```

Comment:
Should be (300px per tile) * 3 + (30px for padding between tiles) * 2 = 960px. Perhaps this value should also be in `core/style/Sizes.js`.

Reply:
Fixed